content/go/concepts/math-functions/terms/modf/modf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'Modf()'
+Description: 'Returns the integer and fractional part of a floating-point number.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Modf()`** function in Go is used to find the integer and fractional part of a floating-point number. This function is part of the `math` package, and the package must be imported to use the function in a Go program.
+
+## Syntax
+
+```pseudo
+result := Modf(f float64) (int float64, frac float64)
+```
+
+Where:
+
+- `f`: is the floating-point number for which you want to extract the integer and fractional parts.
+- `float64`: is a type of numer that uses 64 bytes.
```

Comment:
```suggestion
```

Reply:
lines 28 and 29 should be edited so that they are presented as returns not as parameters

Reply:
Hey @caupolicandiaz, does it sound better if I update it to this?
- `int`: is a returned integer part value.
- `frac`: is a returned fractional part value.